bentoml/saved_bundle/py_module_utils.py

```diff
@@ -148,51 +147,19 @@ def _patch_find_module(name, path=None):
         # with current python version. And that may result in SyntaxError.
         pass
 
-    # extra site-packages or dist-packages directory
```

Comment:
This is no longer needed now, as we've added site packages to the `excludes` list above when creating the modulefinder instance